src/ring_theory/ideal/basic.lean

```diff
@@ -373,6 +375,67 @@ theorem is_prime.pow_mem_iff_mem {I : ideal α} (hI : I.is_prime)
   {r : α} (n : ℕ) (hn : 0 < n) : r ^ n ∈ I ↔ r ∈ I :=
 ⟨hI.mem_of_pow_mem n, (λ hr, I.pow_mem_of_mem hr n hn)⟩
 
+theorem pow_multiset_sum_mem_span_pow (s : multiset α) (n : ℕ) :
+  s.sum ^ (s.card * n + 1) ∈ span ((s.map (λ x, x ^ (n + 1))).to_finset : set α) :=
```

Comment:
Does this generalize at all to
```lean
  s.sum ^ m ∈ span ((s.map (λ x, x ^ n)).to_finset : set α) :=
```
under suitable conditions on n and m? I was able to also show
```suggestion
  s.sum ^ (s.card * n + 1) ∈ span ((s.map (λ x, x ^ n)).to_finset : set α) :=
```
with minimal changes to the proof, which suggests these are both special cases of a more general statement.

Reply:
It's not clear to me whether the `x ^ (n + 1)` statement is stronger or weaker than the `x ^ n` statement

Reply:
The span of `{ f₁ⁿ⁺¹, ..., fᵢⁿ⁺¹ }` should be a subset of the span of `{ f₁ⁿ, ..., fᵢⁿ }` I would suppose?

Reply:
That is to say, for the
```lean
  s.sum ^ m ∈ span ((s.map (λ x, x ^ n)).to_finset : set α) :
```
version, we can currently prove:
* `s.card * n' + 1 ≤ m`, `n = n' + 1` (your lemma + `idea.mul_left_mem`)
* `s.card * n + 1 ≤ m`, `n = n'` (my variant)

Reply:
Oh the catch is probably near `0` 😦
Should I do `m = s.card * (n - 1) + 1` then?

Reply:
> The ideal generated by `{ f₁ⁿ⁺¹, ..., fᵢⁿ⁺¹ }` should be a subset of the ideal generated. by `{ f₁ⁿ, ..., fᵢⁿ }` I would suppose?

Ah yes you're right, I keep confusing ideals with submonoids. In that case, the `+1` as you have it is a stronger statement after all